iFixit/react-commerce
# Add automation bot

This is a second step to address #1017 

The generation of a delegate token is a little bit cumbersome since Shopify doesn't provide any UI to get a delegate token, we need to generate it via API.
To make this convenient, I'm introducing a bot package that will host every little automated script that we might need down the road.

## QA

1. Clone the project and follow the readme setup instructions
2. Run the following command:

    ```console
    pnpm bot shopify create delegate-token
    ```
    Follow the instructions and verify that you can successfully get a delegate token

    ![image](https://user-images.githubusercontent.com/4640135/202430757-918988c9-1998-42e5-8ad9-9abe8e1d5e86.png)

🚨 **Please note that the delegate token should be considered a secret, and thus should not be shared, nor posted on Github.**
